test/test_quantizer.py

```diff
@@ -85,6 +83,7 @@ def get_forward(m):
             return m._c._get_method('forward')
         # TODO: test jit.script as well
         torch._C._jit_pass_constant_propagation(get_forward(script_module).graph)
+        torch._C._jit_pass_constant_propagation(script_module._c._get_module('conv')._get_method('conv2d_forward').graph)
```

Comment:
I'm still not sure why we need to run constant prop.

Reply:
this is just to make checking easier, if we don't do this, we'll see many branches and many observers. Also I haven't test the code with branches yet, so it might break in these cases.

Reply:
Ok, let's keep it for now.